websites/D/Dual/presence.ts

```diff
@@ -0,0 +1,88 @@
+const presence = new Presence({
+  clientId: "741011161079873713"
+});
+
+let sname, sartist, duallisteners, dualislive, dualpresenter;
+
+function metadataListener(): void {
+      const data = JSON.parse(this.responseText);
+      sname = data.now.song;
+      sartist = data.now.artist;
+      duallisteners = data.listeners.current;
+      dualislive = data.presenter.autoDJ;
+      dualpresenter = data.presenter.username;
+}
+
+function updateMetaData(): void {
+  const xhttp = new XMLHttpRequest();
+  xhttp.addEventListener("load", metadataListener);
+  xhttp.open("GET", "https://api.dual.pw/stats", true);
+  xhttp.send();
+}
+
+setInterval(updateMetaData, 10000);
+window.onload = function(): void {updateMetaData()};
+
+let lastTitle,
+	lastTimeStart = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo",
+    smallImageKey: "dualfm-play-v2"
+  },
+  changedetails = await presence.getSetting("changedetails"),
+  changestate = await presence.getSetting("changestate"),
+  changesmalltext = await presence.getSetting("changesmalltext");
```

Comment:
the `|| true` 's are gone?

Reply:
I don’t personally think adding the fallback values for those values are necessary as we’ve not received any issues with those setting values. The only issues we’ve received are regarding the ‘Elapse’ setting.